src/platform/forms-system/src/js/widgets/CheckboxWidget.jsx

```diff
@@ -25,6 +32,7 @@ export default function CheckboxWidget({
         required={required}
         disabled={disabled}
         onChange={event => onChange(event.target.checked)}
+        {...getProps(typeof value === 'undefined' ? false : value)}
```

Comment:
```suggestion
        {...getProps(value ?? false)}
```
We can use the [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator) operator here to clean it up a bit.